### PR TITLE
[Doc] Update broker load faq doc

### PR DESCRIPTION
**Pull request by amber-create**
Signed-off-by: amber-create <yangyanping@starrocks.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: docs/faq/loading/Broker_load_faq.md [hunk 1]
```diff
@@ -1,27 +1,96 @@
...
+)
+```
+
+The preceding example maps the `tmp_c1` and `tmp_c2` columns of the source data file onto the `name` and `id`  columns of the destination StarRocks table, respectively. If you do not specify the `SET` clause, the column names specified in the `column_list` parameter are used to declare the column mapping. For more information, see [BROKER LOAD](../../sql-reference/sql-statements/data-manipulation/BROKER%20LOAD.md).
+
+> **NOTICE**
+>
+> If the source data file is an ORC-formatted file generated by Apache Hive™ and the first row of the file holds `(_col0, _col1, _col2, ...)`, the "Invalid Column Name" error may occur. If this error occurs, you need to use the `SET` clause to specify the column mapping.
+
+## 7. How do I handle errors such as the error due to which the Broker Load job remains running for a long period of time and cannot finish?
```
Comment (evelynzhaojie) [inline docs/faq/loading/Broker_load_faq.md:42]: such as the error that causes the Broker Load job to run for an excessively long period of time and cannot finish?

**Review by evelynzhaojie — DISMISSED** [review]

**Review by EsoragotoSpirit — DISMISSED** [review]

**Comment by amber-create** [comment]
run starrocks_admit_test

**Review by evelynzhaojie — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by amber-create** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.4

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.3

**Pull request merged**